lbplanner/services/user/get_user.php

```diff
@@ -112,7 +112,7 @@ public static function get_user_returns(): external_single_structure {
                 'planid' => new external_value(PARAM_INT, 'The id of the plan the user is assigned to'),
                 'colorblindness' => new external_value(PARAM_TEXT, 'The colorblindness of the user'),
                 'displaytaskcount' => new external_value(PARAM_INT, 'The displaytaskcount of the user'),
-                'capabilities' => new external_value(PARAM_INT, 'The capability'),
+                'capabilities' => new external_value(PARAM_INT, 'The capability of the user'),
```

Comment:
why singular in the docs but plural in the var name?

also add that this is a bitmap of the capabilities